build/common.sh

```diff
@@ -380,19 +380,19 @@ function kube::build::docker_build() {
   local -r context_dir=$2
   local -r pull="${3:-true}"
   local -r build_args=$4
-  local -ra build_cmd=("${DOCKER[@]}" build -t "${image}" "--pull=${pull}" "${build_args}" "${context_dir}")
+  local -ra build_cmd=("${DOCKER[@]}" buildx build --load -t "${image}" "--pull=${pull}" "${build_args}" "${context_dir}")
 
   kube::log::status "Building Docker image ${image}"
   local docker_output
-  docker_output=$("${build_cmd[@]}" 2>&1) || {
+  docker_output=$(DOCKER_CLI_EXPERIMENTAL=enabled "${build_cmd[@]}" 2>&1) || {
```

Comment:
I can't seem to figure out a way to move DOCKER_CLI_EXPERIMENTAL=enabled to line 383, I tried setting line 383 to 
```
  local -ra build_cmd=("DOCKER_CLI_EXPERIMENTAL=enabled" "${DOCKER[@]}" buildx build --load -t "${image}" "--pull=${pull}" "${build_args}" "${context_dir}")
```
and leaving line 387 unchanged but I get the following error: 
```
build/../build/common.sh: line 387: DOCKER_CLI_EXPERIMENTAL=enabled: command not found
To retry manually, run:

DOCKER_CLI_EXPERIMENTAL=enabled DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --load -t kube-build:build-7ac69468fc-5-v1.15.8-1 --pull=false --build-arg=KUBE_BUILD_IMAGE_CROSS_TAG=v1.15.8-1 /<redacted>/k8s.io/kubernetes/_output/images/kube-build:build-7ac69468fc-5-v1.15.8-1

!!! [0214 12:56:06] Call tree:
!!! [0214 12:56:06]  1: build/release.sh:35 kube::build::build_image(...)
make: *** [Makefile:452: quick-release] Error 1
```

Reply:
we can `export DOCKER_CLI_EXPERIMENTAL=enabled` this in common.sh probably, I don't think we have a huge reason not to. Some of our makefiles just do this